volmdlr/curves.py

```diff
@@ -2186,6 +2186,8 @@ def ellipse_intersections(self, ellipse, abs_tol: float = 1e-6):
         """
         intersections = []
         # from volmdlr import surfaces
+        # plane1 = surfaces.Plane3D(self.frame)
+        # plane2 = surfaces.Plane3D(ellipse.frame)
```

Comment:
Clean?

Reply:
Okay. I'll clean it in the one coming ahead.